src/commands/schema_diff.ts

```diff
@@ -192,6 +192,11 @@ export const parseSchemaDiffParams = (props: SchemaDiffProps) => {
         `No branches specified. Comparing branch '${props.branch}' with its parent`,
       );
       props.compareSource = '^parent';
+    } else {
```

Comment:
What if the primary branch is the root branch (I think that's valid for most projects)?
I'd suggest to make it required to pass at least one branch

Reply:
On that case we do display messages saying there is no primary branch.

Users can rerun the command with new arguments.

Reply:
but there is a primary branch...

Reply:
sorry, I mean there is no parent branch to primary branch

Reply:
Updated the logic to verify the primary and context branches have parents before comparing.